service/src/tests/mock.rs

```diff
@@ -8,8 +8,8 @@ pub const W2_URL: &str = "127.0.0.1:3333";
 
 pub fn enclaves() -> Vec<Enclave> {
 	vec![
-		Enclave::new([0; 32].into(), [1; 32], 1, format!("ws://{}", W1_URL)),
-		Enclave::new([2; 32].into(), [3; 32], 2, format!("ws://{}", W2_URL)),
+		Enclave::new([0; 32].into(), [1; 32], 1, format!("wss://{}", W1_URL)),
+		Enclave::new([2; 32].into(), [3; 32], 2, format!("wss://{}", W2_URL)),
 	]
 }
```

Comment:
I think if unit tests would actually use this, they would break, as we can't talk over wss locally.

Reply:
I disagree here: We're talking via wss locally: 
https://github.com/integritee-network/worker/blob/3376deb0c15357f2339c668087ded589e7388a94/cli/src/main.rs#L95-L102

unless set differently, wss is taken as default value. And I don't think this is set anywhere in the CI nor am I setting it locally..?

Reply:
Ok, fine. I am convinced. Happy to see that this code is used. :)